### DISCUSSION TITLE
Thanks for the 0.8.0 release!

### DISCUSSION THREAD

**Discussion by apikee** [pocketbase/pocketbase#1066] [Q&A]
I just wanted to thank you for the new release and documentation improvements. Good work and I look forward to further development. Btw. what are the next plans? It's not entirely clear from the roadmap what you're going to do next. So what can we look forward to in 0.9.0? 

Just a nitpicky note - in the documentation (https://pocketbase.io/docs/api-records/) there is probably still an example from the previous version at Create/Update/Delete Record, `client.records.create('demo'...`.

**Comment by ganigeorgiev** [comment]
Thank you, I've updated the examples (btw. the site documentation now is available in https://github.com/pocketbase/site without the semi-automated part so if you find some typos or think that something could be explained better, feel free to open a PR in the site repo).

>  It's not entirely clear from the roadmap what you're going to do next. So what can we look forward to in 0.9.0?

There is no strictly fixed deadline and definition what will land in the next v0.9.0, but I'll try to focus on the things that are in the ["In progress" column](https://github.com/orgs/pocketbase/projects/2/views/1):

- Research goja to allow writing migrations with JS/TS when the prebuilt PocketBase binary is used.
- New event hooks per https://github.com/pocketbase/pocketbase/issues/468 (and maybe more related to the mail sending operations).
- Filter enhancement - allow support for querying relation fields through `@request.data.*`, "IN" style lookups and multi-match expressions (eg. `members.status *= "pending"` should result to true only when all multi-relation `members` records have status = "pending"; I plan to create a dedicated discussion for this in the near future before deciding on the final operators syntax)
